### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by rheber** [Ceasar/twosheds#48]
Currently relies on readline, which "is not meant to work on Windows".

**Comment by Ceasar** [comment]
Yeah, this would be good if it's not much trouble. It is not an obvious priority of mine since I use a Mac, but if you want to add it looks like we might just need to add another dependency: http://stackoverflow.com/questions/6024952/readline-functionality-on-windows-with-python-2-7
